### DISCUSSION TITLE
I am trying to follow 6.1 model, but "784" and "28 * 28" works totally different.

### DISCUSSION THREAD

**Discussion by Feiyang-umich** [mrdbourke/pytorch-deep-learning#1132] [Q&A]
`model_01 = FashionMNISTv1(784,10,len(class_names)).to(device)`  works the same as the tutorial
while `model_01 = FashionMNISTv1(28*28,10,len(class_names)).to(device)` only gives  ~10% percent accuracy with loss ~ 2.3.

Any difference between these styles?

**Comment by LuluW8071** [comment]
There is no difference, its basically same. As its just simpler way of readability for $28$ * $28$. Easier to know its $784$ comes from height * width.
